src/sciline/visualize.py

```diff
@@ -87,7 +87,7 @@ def get_subgraph_name(name: str, kind: str) -> str:
         if kind == 'series':
             # Example: Series[RowId, Material[Country]] -> RowId, Material[Country]
             return name.partition('[')[-1].rpartition(']')[0]
-        return name
+        return name.split('[')[0]
```

Comment:
Where/how was this actually broken?

Reply:
What about this discussion, where you pointed out something related? https://github.com/scipp/sciline/pull/124/files#r1489495340

Reply:
It never made clusters, it always returned `RawData[Sample]` and `RawData[Background]` instead of `RawData` for both which would then make a cluster, I think?